lib/router/handler-info/unresolved-handler-info-by-object.js

```diff
@@ -34,8 +35,8 @@ var UnresolvedHandlerInfoByObject = subclass(HandlerInfo, {
     }
 
     // Use custom serialize if it exists.
-    if (handler.serialize) {
-      return handler.serialize(model, names);
+    if (serializer) {
+      return serializer(model, names);
```

Comment:
This introduced a breaking change. Any route that implemented the ```serialize``` hook and referenced ```this``` from within the hook is now broken, since the serialize method is called unbound. Maybe too late to do anything about it, and I guess that pattern is on its way out anyway, but thought I'd mention it so folks are aware.